__tests__/client/actions/agency.js

```diff
@@ -0,0 +1,31 @@
+/* global describe */
+
+import {makeGenericModelActionsTests} from '../../test-utils/actions'
+import {blankAgency} from '../../test-utils/mock-data'
+
+import * as agency from '../../../client/actions/agency'
+
+describe('actions > agency', () => {
+  makeGenericModelActionsTests({
```

Comment:
For something as common as REST endpoints, I like this configuration because it is consistent and much less code to screw up.  Furthermore, it is entirely possible to only configure some but not all of the rest endpoints.